cyprecice/cyprecice.pyx

```diff
@@ -1104,6 +1109,8 @@ cdef class Interface:
             ID to read from.
         vertex_id : int
             Index of the vertex.
+        relative_read_time : double
+            Point in time where data is read relative to the beginning of the current time step
```

Comment:
What should a user do if they want a relative time of 0?
Pass 0, pass None, or omit the argument?

Reply:
Pass `relative_read_time=0`. This points to the beginning of the current time *step*. The API is designed such that you always get the value at the end of the window, if you pass nothing (or `None`), because this is the current behavior of preCICE. This is equivalent to calling a read function with `relative_read_time = dt`, where `dt = precice.advance(...)`. Does this help?

Reply:
That helps me, but not the reader of the documentation :wink:

Reply:
I will finalize this for now, let us revisit the documentation if there is indeed confusion on the user side. We want to release this API for a first user, so we shall know then.

Reply:
@IshaanDesai Documentation of the CPP API is much more detailed. See [here](https://github.com/precice/precice/blob/fcefb9d6620f8d4f8dfefa574bbcd44002ce7adf/src/precice/SolverInterface.hpp#L880-L910). I guess keeping this synchronized is indeed a different issue. Ideally this would happen automatically, but I know... Let's not worry about this here.